lib/assessment.rb

```diff
@@ -21,13 +23,14 @@ def sum(enum)
   # in the numbers enumerable are odd
   def some_odd(numbers)
     # Your code goes here
+    numbers.reduce(false) {|memo, item| memo || item.odd?}
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
   def every_even(numbers)
-    # Your code goes here
+    numbers.reduce(true) {|memo, item| memo && item.even?}
```

Comment:
See above. Correct answer, but best practice would probably be something like 
`numbers.all? {|num| num.even?}`
or
`numbers.all?(&:even?)`